scripts/globals/weaponskills.lua

```diff
@@ -337,6 +337,7 @@ local function getSingleHitDamage(attacker, target, dmg, wsParams, calcParams, f
                 local ftpHybrid = xi.weaponskills.fTP(calcParams.tp, wsParams.ftp100, wsParams.ftp200, wsParams.ftp300) + calcParams.bonusfTP
                 local magicdmg = finaldmg * ftpHybrid
 
+                wsParams.ignoreStateLock = true
```

Comment:
ignoreStateLock isnt a thing - this should of been removed. The issue is that these are following the enfeebling calculations by accident. If we use `params.damageSpell = true` that should fix them all. An oversight on my end.

Reply:
Converted all uses of ignorestatelock to damageSpell
Removed ignorestatelock

Noticing an oddity that may be explainable - but I do not fully understand the following

Level 60 smn - 196 staff skill.  Nekkid w/ light staff.
Target: Giant Buffalo level 68
Weaponskill - rock crusher

I am only able to get 1/2 resists on Rock Crusher.
I expect to get no, 1/2, 1/4, and 1/8 resists.

Debug output:
![image](https://github.com/AirSkyBoat/AirSkyBoat/assets/105882290/fd6172fe-782e-44a4-a01a-30f416705f0f)

Same seems to happen for level 60 shiva Blizzard 4.

Reply:
I basically read the code as 
```
if the target does not have MEVA over 101 or T11 EEM - it can not 1/8th resist
if the target does not have MEVA between 0 and 100 - it can not 1/4 resist
Therefore - since the buffalo has no earth MEVA - the best it can do is half resist.
```

Is this this the intented behavior?

Reply:
Can you check if params.damageSpell is returning true inside the applyAbilityResistance

Reply:
Can reverify tonight (omw to the office today)
damageSpell was true
restTriggerPoints[1] and [2] were not true - therefore eigthTrigger and quaterTrigger were not true - therefore on the rolls - 1/8 and 1/4 resists were not possible

![image](https://github.com/AirSkyBoat/AirSkyBoat/assets/105882290/331ca0d0-280e-4f03-9fac-3d23475a40e5)

Reply:
This all ties back to resMod (or in this case EARTH_MEVA) being 0, and eem not being T11.
The buffalo is essentially neutral to earth - (not weak, not strong) and it therefore via code cannot produce a half or quarter as far as i can tell.

Reply:
after much discussion - we've enabled 1/4 resists vs mobs 
Wil continue to dig into this and update in future PRs as we learn more